### ISSUE TITLE
Android Q(10) Error. error=13, Permission denied

### ISSUE THREAD

**Issue by aliraza96** [adrielcafe/AndroidAudioConverter#37]
Since Android 10, this error is coming when trying to convert

<img width="1072" alt="Screenshot 2020-05-12 at 12 53 09 PM" src="https://user-images.githubusercontent.com/59387704/81655249-89982c00-944f-11ea-86da-37458235cffb.png">

Can anyone help me with this?

**Comment by rafagan** [comment]
I'm also experiencing the same problem

**Comment by jhui20130703** [comment]
I encountered the same problem as you, have you solved it?

**Comment by rafagan** [comment]
No, I’ve integrated ffmpeg in ndk and did the conversion by my own

Ráfagan Abreu
________________________________

**Comment by PDharam** [comment]
I faced the same issue. Have you planned to resolved it.

**Comment by Wisdozzh** [comment]
From Android Q onwards, you cannot execute binaries in your app's private data directory.
From the issuetracker: https://issuetracker.google.com/issues/128554619

Change only on Build.gradle file targetSdkVersion 29 to 28 and Re-Install your app on your device - It is resolved your permission issue for temporary because of the targetSdkVersion 29 is required platform for released build on play store so I suggest to you use this library
